### PR TITLE
About Us Page updated

### LINKED ISSUE #346: [Feature Request]: Proper UI and Interactivity for the entire About Us Page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

The About Us Page has no interactive and sleek animations and does not have a proper UI, there are less gaps and less spacings and the texts are almost overlapping in smaller devices. 

### Use Case

Proper Ui ensures better User Experience and improves readability of the website.

Sleek animations and lottie files will make the Styleshare more interactive and fun to visit.

### Benefits

_No response_

### Add ScreenShots

Screenshot 2024-07-01 005223
Screenshot 2024-07-01 005733

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by r7projects-shayan**
# Pull Request

### Title
Updated the About Us page 

### Description

Updated the About Us page and made it fully responsive for all devices and dimensions
Added Interactive and sleek animations
Fixed the spacings

### Related Issues

Fixes #346 

### Changes Made

New Features and bug fixes

### Checklist 

- [x] I have tested the changes locally
- [x] Documentation has been updated (if necessary)
- [x] Changes are backward-compatible

## Video Demo

https://github.com/user-attachments/assets/a6ecd3ed-670c-49bb-a691-a2cb4c39dade

## These are the screenshots

Screenshot 2024-07-14 202625
Screenshot 2024-07-14 202632
Screenshot 2024-07-14 202644
Screenshot 2024-07-14 202648
Screenshot 2024-07-14 202703
Screenshot 2024-07-14 202710
Screenshot 2024-07-14 202716
Screenshot 2024-07-14 202744
Screenshot 2024-07-14 202749
Screenshot 2024-07-14 202755
Screenshot 2024-07-14 202800
Screenshot 2024-07-14 202804

### Additional Notes

Footer

### CODE REVIEW

**Comment by r7projects-shayan** [comment]
just give me some time I need to see the deployment error by deploying it on my personal vercel

**Comment by r7projects-shayan** [comment]
Yes done now updated successfully

**Review by VaibhavArora314 — CHANGES_REQUESTED** [review]
It looks great, just have one suggestion that is whole website is using tailwind throughout and I don't think we should have another library for styling even though it is built top on tailwind.

**Comment by VaibhavArora314** [comment]
Also the how it works options should always be selected, we are not allowing users to select them rather we are showing what we offer and how website works

**Comment by r7projects-shayan** [comment]
@VaibhavArora314 Alright I'll remove the checkbox interactivity and remove the external styling library

**Review by Ultimateutkarsh11 — APPROVED** [review]

File: frontend/src/styles/about.css
Comment (VaibhavArora314) [inline frontend/src/styles/about.css:1]: remove commented code here

**Review by VaibhavArora314 — COMMENTED** [review]
looks great, I have one question I wanted to ask what is lottieabout.json for?

**Comment by r7projects-shayan** [comment]
@VaibhavArora314 the lottie.json is for the animation in the about us page in place of the image

File: frontend/src/styles/about.css
Comment (r7projects-shayan) [reply frontend/src/styles/about.css:1]: done

**Review by VaibhavArora314 — APPROVED** [review]

**Comment by akbatra567** [comment]
@r7projects-shayan, resolve conflicts.

**Review by akbatra567 — APPROVED** [review]

**Pull request closed without merging**